lib/application.js

```diff
@@ -44,37 +45,53 @@ Application.prototype.setupPromiseness = function () {
 
 Application.prototype.start = function () {
   var self = this
-  return self.exists()
-    .then(function () { return self.startChromeDriver() })
-    .then(function () { return self.createClient() })
-    .then(function () { return self.api.initialize() })
-    .then(function () { return self.client.timeouts('script', self.waitTimeout) })
-    .then(function () { self.running = true })
-    .then(function () { return self })
+  return self
+    .exists()
+    .then(function () {
+      return self.startChromeDriver()
+    })
+    .then(function () {
+      return self.createClient()
+    })
+    .then(function () {
+      return self.api.initialize()
+    })
+    .then(function () {
+      return self.client.setTimeouts(
+        self.waitTimeout,
+        self.waitTimeout,
+        self.waitTimeout
+      )
+    })
+    .then(function () {
+      self.running = true
+    })
+    .then(function () {
+      return self
+    })
```

Comment:
instead of using `self` we could just use short functions:
```suggestion
  return this
    .exists()
    .then(() => this.startChromeDriver())
    .then(() => this.createClient())
    .then(() => this.api.initialize())
    .then(() => this.client.setTimeouts(this.waitTimeout, this.waitTimeout, this.waitTimeout))
    .then(() => {
      this.running = true
    })
    .then(() => this)
```